### PR TITLE
Fix clear session

### PR DESCRIPTION
**Pull request by koen92**
This PR will fix a bug that lets the frontend 'hang' in 'Disclosure successful' state when the entire browser session is cleared (the clearSession action).

This is fixed by clearing up the entire divaSession after we stop polling. Because in this way, divaSession won't longer always exist, we also needed to fix the three views by checking on existence of divaSession before checking divaSession.sessionStatus.

I also removed the 'isPoling' state from the reducer, since that variable is never used anywhere?

I'm still not completely convinced that my fixes will cover all the edge cases, by not introducing new bugs. So please think about it, check and test carefully before merging this PR.

### CODE REVIEW

File: src/lib/reducers/diva-reducer.js [hunk 1]
```diff
@@ -57,7 +57,6 @@ export default (state = initialState, action) => {
           ...state.sessions,
           [action.viewId]: {
             irmaSessionType: action.irmaSessionType,
-            isPolling: false,
```
Comment (timenolthof) [inline src/lib/reducers/diva-reducer.js:60]: Is there no case at all where the `isPolling` flag is useful in UI's? Or can it be derived from other props?

**Review by timenolthof — APPROVED** [review]

**Comment by koen92** [comment]
Won't be fixed since we are migrating to new IRMA server
